Modules/Utils/Sources/PocketCastsUtils/Logging/FileLog.swift

```diff
@@ -102,36 +128,26 @@ public class FileLog {
         .eraseToAnyPublisher()
     }
 
-    private func appendStringToLog(_ line: String) {
+    private func writeLogBufferToDisk() {
+        let logTimestampFormatter = DateFormatHelper.sharedHelper.localTimeJsonDateFormatter
+        let newLogChunk = logBuffer.reduce(into: "") { resultChunk, logEntry in
+            let formattedTimestamp = logTimestampFormatter.string(from: logEntry.timestamp)
+            let logLine = "\(formattedTimestamp) \(logEntry.message)\n"
```

Comment:
Maybe we can move this formatation logic to `LogEntry`?

Reply:
Good idea – the `DateFormatter` is static so it's nice isolating it to `LogEntry`. Updated ✅